### ISSUE TITLE
Support capturing PURGE requests

### ISSUE THREAD

**Issue by calvinhp** [jbittel/httpry#15]
I'm trying to debug an issue with our caching servers and `httpry` doesn't appear to be able to grab `PURGE` requests even if I specify it using the `-m` flag.

**Comment by jbittel** [comment]
Interesting...it should be parsing them if you've specified that method. Can you provide the full command-line invocation that you're using, and if possible, a pcap file containing at least one PURGE request?

Thanks!

**Comment by calvinhp** [comment]
I pulled the latest code from github and compiled it from source and it is working as expected.  The package I was using was from FreeBSD's pkg system and was version 0.1.7.  I think this may be an invalid bug as it works with the source checkout.

Actually, it appears that the packaged version with FreeBSD doesn't properly work against anything on the loopback interface.  I'll see if I can open a bug with the port maintainer. My guess is the system libpcap isn't working right, but I compiled yours against one I installed from packages and this one appears to work as expected.  Sorry for the noise and thanks for the quick response.
